src/test/http-mocks/fees.ts

```diff
@@ -2,89 +2,157 @@ import * as config from 'config'
+        },
+        author: 'LOADER',
+        approvedBy: 'LOADER'
+      }
+    ],
+    current_version: {
+      version: 1,
+      description: 'Civil Court fees - Money Claims Online - Claim Amount - 0.01 upto 300 GBP',
+      status: 'approved',
+      flat_amount: {
+        amount: 25
+      },
+      author: 'LOADER',
+      approvedBy: 'LOADER'
+    },
+    min_range: 0,
+    max_range: 300,
+    range_unit: 'GBP',
+    unspecified_claim_amount: false
+  }
+]
 
 export function resolveCalculateIssueFee (): mock.Scope {
-  return resolveCalculateFee(issueFeeCode)
+  return resolveCalculateFee(issueEvent, onlineChannel)
 }
 
 export function rejectCalculateIssueFee (reason: string = 'HTTP error'): mock.Scope {
-  return rejectCalculateFee(issueFeeCode, reason)
+  return rejectCalculateFee(issueEvent, reason, onlineChannel)
 }
 
 export function resolveCalculateHearingFee (): mock.Scope {
-  return resolveCalculateFee(hearingFeeCode)
+  return resolveCalculateFee(hearingEvent, defaultChannel)
 }
 
 export function rejectCalculateHearingFee (reason: string = 'HTTP error'): mock.Scope {
-  return rejectCalculateFee(hearingFeeCode, reason)
+  return rejectCalculateFee(hearingEvent, reason, defaultChannel)
 }
 
 export function resolveGetIssueFeeRangeGroup (): mock.Scope {
-  return resolveGetFeeRangeGroup(issueFeeCode)
+  return resolveGetFeeRangeGroup(issueEvent, onlineChannel)
 }
 
 export function rejectGetIssueFeeRangeGroup (reason: string = 'HTTP error'): mock.Scope {
-  return rejectGetFeeRangeGroup(issueFeeCode, reason)
+  return rejectGetFeeRangeGroup(issueEvent, reason, onlineChannel)
 }
 
 export function resolveGetHearingFeeRangeGroup (): mock.Scope {
-  return resolveGetFeeRangeGroup(hearingFeeCode)
+  return resolveGetFeeRangeGroup(hearingEvent, defaultChannel)
 }
 
 export function rejectGetHearingFeeRangeGroup (reason: string = 'HTTP error'): mock.Scope {
-  return rejectGetFeeRangeGroup(hearingFeeCode, reason)
+  return rejectGetFeeRangeGroup(hearingEvent, reason, defaultChannel)
 }
 
-export function resolveCalculateFee (code: string): mock.Scope {
-  return mock(serviceBaseURL)
-    .get(new RegExp(`/${code}/calculations\\?value=[0-9]+`))
-    .reply(HttpStatus.OK, calculationOutcome)
+export function resolveCalculateFee (eventType: string, channel: string): mock.Scope {
+  return mock(baseFeeUri)
+    .get(`/fees-register/fees/lookup`)
+    .query({
+      service: `${service}`,
+      jurisdiction1: `${jurisdiction1}`,
+      jurisdiction2: `${jurisdiction2}`,
+      channel: `${channel}`,
+      event: `${eventType}`,
+      amount_or_volume: new RegExp(`[\\d]+`)
+    })
+    .reply(HttpStatus.OK, feeOutcome)
 }
 
-export function rejectCalculateFee (code: string, reason: string = 'HTTP error'): mock.Scope {
-  return mock(serviceBaseURL)
-    .get(new RegExp(`/${code}/calculations\\?value=[0-9]+`))
+export function rejectCalculateFee (eventType: string, reason: string = 'HTTP error', channel: string): mock.Scope {
```

Comment:
I would love to have the following order:
- eventType: string
- channel: string
- reason: string = 'HTTP error'